packages/cli/src/cog/cog.stac.job.ts

```diff
@@ -284,9 +285,11 @@ export class CogStacJob implements CogJob {
         for (const f of covering.features) {
             const { name } = f.properties as { name: string };
             const href = name + '.json';
+            const bbox = f.bbox as BBox;
             links.push({ href, type: 'application/geo+json', rel: 'item' });
             const item: CogStacItem = {
                 ...f,
+                bbox,
```

Comment:
cause; typescript

Reply:
It would imply there is a structural difference between the two types? what is the difference

Reply:
bbox if optional for GeoJSON but compulsory for STAC

Reply:
I could make a new interface with a compulsory `bbox` for Covering files and use that instead of `GeoJSON`? At least that we I wont be executing code the get around the type difference.

Reply:
Added new type `BBoxFeature`